testsuite/MDAnalysisTests/coordinates/test_trz.py

```diff
@@ -129,6 +129,10 @@ def test_get_writer_2(self, universe, tmpdir):
             assert_equal(isinstance(W, mda.coordinates.TRZ.TRZWriter), True)
             assert_equal(W.n_atoms, 100)
 
+    def test_get_wrong_n_atoms(self):
+        with pytest.raises(ValueError, match=r"Supplied n_atoms *"):
```

Comment:
note,  I dont think you need to have the "*" in match calls, it essentially does a `string in string` type of comparison.